wikimedia/wikipedia-ios
# Rename the Abkhaz language

**Phabricator:** 
https://phabricator.wikimedia.org/T298309

### Test Steps
1. Open the English Wikipedia article "Fazil Iskander".
2. Check the interlanguage links. The name of the language whose code is "ab" is supposed to be "аԥшәа". The first letter is supposed to be lowercase, and the second letter is supposed to have a tiny tail (ԥ) and not a large hook (ҧ).
